pkgs/os-specific/linux/ply/default.nix

```diff
@@ -3,7 +3,7 @@
 assert kernel != null -> lib.versionAtLeast kernel.version "4.0";
 
```

Comment:
```suggestion
```

Reply:
The assertion condition should be assigned to `meta.broken` so.